bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyDeviceProfile.java

```diff
@@ -331,7 +332,7 @@ public boolean inButtonMode(int idx) {
         } else if (isDimmer) {
             if (settings.dimmers != null) {
                 ShellySettingsDimmer dimmer = settings.dimmers.get(0);
-                btnType = dimmer.btnType;
+                btnType = getString(dimmer.btnType);
```

Comment:
I saw this one time during debugging the new Plus Dimmer, therefore I added this check. It's added just for safety and to avoid an NPE in case of...

Reply:
Does it only affect the new devices, or is it a general bugfix?

Reply:
I saw this while adding support for the new Dimmer, I didn't spent time to validate others

Reply:
I have extracted it to #17180, so that it can be backported, and we can potentially avoid `NullPointerExceptions` for other dimmers or lights as well.